modules/rosa-delete-objects.adoc

```diff
@@ -109,6 +109,55 @@ Delete a cluster named `mycluster`.
 $ rosa delete cluster --cluster=mycluster
 ----
 
+[id="rosa-delete-external-auth-provider_{context}"]
+== delete external-auth-provider
+
+Deletes an external authentication provider from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete external-auth-provider <name_of_external_auth_provider> --cluster=<cluster_name> | <cluster_id> [arguments]
+----
+
+.Arguments
```

Comment:
Those are options, you don't actually describe the accepted argument (`name_of_external_auth_provider`) in this section.

Reply:
Is that the case for all of the ROSA CLI? I was mirroring the other existing sections. I think we picked "arguments" because some of the options were required while some where optional.

https://file.rdu.redhat.com/eponvell/OSDOCS-9614_External-OIDC-Auth-HCP/cli_reference/rosa_cli/rosa-manage-objects-cli.html